delivery_free_fee_removal/reports/sale_report_templates.xml

```diff
@@ -5,7 +5,7 @@
         inherit_id="sale.report_saleorder_document"
     >
         <xpath
-            expr="//tbody[hasclass('sale_tbody')]/t[@t-as='line']/tr"
```

Comment:
Is this necessary? this block of code in the inherited template remains the same it doesn't change across versions so I wouldn't consider it part of the migration